components/list/list-item-mixin.js

```diff
@@ -81,10 +82,20 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 				border-bottom: 1px solid transparent;
 				border-top: 1px solid transparent;
 			}
-			[slot="content"].d2l-list-item-content-extend-separators {
+			.d2l-list-item-content-extend-separators > [slot="control"] {
+				width: 3rem;
+			}
+			.d2l-list-item-content-extend-separators > [slot="content"] {
 				padding-left: 0.9rem;
 				padding-right: 0.9rem;
 			}
+			:host([selectable]) .d2l-list-item-content-extend-separators > [slot="content"] {
```

Comment:
This (along with the change further below to add he whitespace to the outside edge of the checkbox) fixes the issue where the `0.9rem` extra whitespace was incorrectly added in between the content and the checkbox when separators are extended.